### PR TITLE
Removing base site url

### PR DESCRIPTION
**Pull request by ajsb85**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 source 'https://rubygems.org'
 gem 'github-pages'
+gem 'wdm'
```
Comment (muescha) [inline Gemfile:3]: you are sure to add the `wdm` gem?

**Comment by tristen** [comment]
Why would we do this?

**Comment by ajsb85** [comment]
To use  _--watch_ on Window local environment.

``` bash
bundle exec jekyll serve --watch
```

You can see the complete commit 7542ccc5c9c40f1a9c07f5bfff4441f27aa0ecb9 

Try remove the _site.url_ var to work locally.

**Comment by ajsb85** [comment]
OTP: I make this addon for Bracket:

https://github.com/alexsalas/alexsalas.brackets-jekyll
https://github.com/adobe/brackets/

**Comment by kostajh** [comment]
@alexsalas you could instead do:

`jekyll serve --baseurl=""`

**Comment by ajsb85** [comment]
@kostajh Oh thank you!
I'll try.

**Comment by struex** [comment]
hey thanks @kostajh that got my local starter fork running!

**Pull request closed without merging**